### PR TITLE
Remove AWS install from ewings action

### PR DESCRIPTION
**Pull request by jashapiro**
I realized while reviewing another PR that the Ewings module already does have `aws` installed, just in the conda environment. So here I am eliminating the install step and substituting activation.

I may do some additional testing, because the Docker image _should_ auto-activate, but github may be overriding that somehow.

### CODE REVIEW

**Comment by jashapiro** [comment]
Converted back to a draft as #591 may affect the implementation here, at least in part.

File: .github/workflows/run_cell-type-ewings.yml [hunk 1]
```diff
@@ -27,22 +27,15 @@ jobs:
...
-      - name: Install awscli
-        run: |
-          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-          unzip awscliv2.zip
-          ./aws/install
-
       - name: Download test data
-        run: ./download-data.py --test-data --format "sce,anndata" --samples SCPCS000490
+        run: |
+          conda activate openscpca-cell-type-ewings
```
Comment (allyhawkins) [inline .github/workflows/run_cell-type-ewings.yml:37]: Do we still need this and the one in the next step if it's activated in the image?

Comment (jashapiro) [reply .github/workflows/run_cell-type-ewings.yml:37]: Hopefully not. Will be testing once the docker image finishes building

**Review by allyhawkins — APPROVED** [review]
LGTM

**Pull request merged**